sdf/1.8/navsat.sdf

```diff
@@ -0,0 +1,40 @@
+<element name="navsat" required="0">
```

Comment:
Should we copy it to 1.9?

Reply:
Thanks for catching this. There were a few more changes that needed to be copied over to the 1.9 directory. I force-pushed since this will be a merge commit.